cmd/owl/main.go

```diff
@@ -196,6 +203,10 @@ func main() {
 
 		jobs := make(chan owl.Job, 10)
 
+		jobs <- &WatcherJob{
```

Comment:
Also comment this line so everyone knows we need this to start the command at the start of the owl